components/CreateOrganization.js

```diff
@@ -149,10 +120,10 @@ class CreateOrganization extends React.Component {
                   onChange={this.resetError}
                 />
 
-                <div className="result">
-                  <div className="success">{this.state.result.success}</div>
-                  <div className="error">{this.state.result.error}</div>
-                </div>
+                <Container textAlign="center" marginBottom="5rem">
+                  <Container style={{ color: 'green' }}>{this.state.result.success}</Container>
+                  <Container style={{ color: 'red' }}>{this.state.result.error}</Container>
```

Comment:
It's better to use the `css` prop when you want to add styles to a styled component (see https://styled-components.com/docs/api#css-prop). However, the `Container` component has support for the `color` prop, and we can even use the design system directly from there.

```suggestion
                  <Container color="green.500">{this.state.result.success}</Container>
                  <Container color="red.500">{this.state.result.error}</Container>
```

Reply:
@Betree : Thanks for the suggestion. I've corrected this.